litmus-portal/frontend/src/components/CustomResultModal/styles.ts

```diff
@@ -0,0 +1,87 @@
+import { makeStyles } from "@material-ui/core";
+
+export const useStyles = makeStyles((theme) => ({
+	testHeading: {
+		marginTop: theme.spacing(6.25),
+		fontFamily: "Ubuntu",
+		fontSize: 25,
```

Comment:
Either use rem values for fontSizes or use Typography component if you don't need a custom fontsize. If you use Typography don't override the fontsize with css.

Reply:
-[x] Fixed